recipes/mprod-package/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "mprod-package" %}
+{% set version = "0.0.1a1" %}
+{% set sha256 = "7b2c969d98bb8e266f168a9ac5ceaba80127d87890162eb0f5f4f48c3f5eace7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mprod-package-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  skip: true  # [win]
```

Comment:
Noarch should not have any arch restrictions. If this is failing on windows bc of a missing pkg it is fine.

Reply:
So deleting line 17 altogether should work?

Reply:
Yep. It may fail on windows but if that is due to a missing pkg it is fine. This one can still be noarch and probably installable in the future if the missing windows pkg is added.

Reply:
Hey @ocefpaf , is there any chance that you approve the last changes so we can move forward with the staging ?

Reply:
> Hey @ocefpaf , is there any chance that you approve/disapprove the last changes so we can move forward with the staging ?

@dopplershift was your original reviewer here. I'll let him finish this. (Actually I did not notice you already had a reviewer when I commented! That's what I get for trying to work from a cellphone.)

Reply:
Thanks for our help @ocefpaf . I'll wait for @dopplershift s response then...